### ISSUE TITLE
Replace PNG sponsor assets with SVGs

### ISSUE THREAD

**Issue by MattIPv4** [Hacktoberfest/hacktoberfest-2022#70]
### Feature description

Replace all the PNGs found in https://github.com/Hacktoberfest/hacktoberfest-2022/tree/main/src/assets/img/partners with SVG equivalents.

### How the feature is useful

Better fidelity of the assets.

**Comment by Siddharthgup** [comment]
Hi @MattIPv4 ,Can you assigned this task to me ?

**Comment by MattIPv4** [comment]
Sure 👍

**Comment by Anandprafull** [comment]
Please check the PR i have replaced the assets as well as the related sponser.js code

**Comment by Anandprafull** [comment]
I see but since there hadn't been any update on the issue so i worked on it..

**Comment by MattIPv4** [comment]
Yup, but it has only been three days since they asked for it to be assigned to them, and only a day since Hacktoberfest started. It is important to communicate on issues, especially where they are assigned to someone else, not just assume you can do it instead.

I'll leave your PR up for now, but there are issues with it that need to be addressed.

**Comment by MattIPv4** [comment]
@Siddharthgup Are you intending to work on this still?

**Comment by Siddharthgup** [comment]
Yes,It's almost completed

**Comment by Lokeshsuwalka05** [comment]
Hello,if it's still open.
Can you assign this issue to me??

**Comment by MattIPv4** [comment]
@Lokeshsuwalka05 as you can see from the issue itself, it is already assigned to someone else.

**Comment by kingsmen732** [comment]
is this issue still open or does it need to closed? @MattIPv4
kindly assign me if its still open :)
